utils/_context/_scenarios/endtoend.py

```diff
@@ -349,8 +349,8 @@ def __init__(
         self.backend_interface_timeout = backend_interface_timeout
         self.library_interface_timeout = library_interface_timeout
 
-    def is_part_of(self, declared_scenario):
-        return declared_scenario in (self.name, "EndToEndScenario")
+    def is_part_of(self, declared_scenarios: list[str]) -> bool:
+        return self.name in declared_scenarios or "EndToEndScenario" in declared_scenarios
```

Comment:
I don't understand this. Why the "or" clause?

Reply:
It for this particular use case : https://github.com/DataDog/system-tests/blob/main/tests/test_schemas.py#L10

Which will be removed soon by #3754 ...